### ISSUE TITLE
Add opposite to `no-anonymous-default-export` rule that will force an error if you name default export

### ISSUE THREAD

**Issue by T-Damer** [import-js/eslint-plugin-import#2456]
Naming `export default function do() {...}` in `do.ts` is not necessary for my team, so maybe we can have a rule, that will throw an error if you're trying to name the default export? I know that this might not be a good practice, but if I'll combine this rule with [matching filename import](https://github.com/selaux/eslint-plugin-filenames#matching-exported-values-match-exported), then nobody will import it wrongly. My team and I don't want to grep thought the repo right now

**Comment by ljharb** [comment]
Why would you want to avoid it? Functions should always be explicitly named, for debugging and greppability.

**Comment by T-Damer** [comment]
@ljharb, the file name in my team usually represents the default export name, we're not planning to grep through our repos + debugging is fine, you can see the file, where the error occurred = you know the function, where the error occurred

**Comment by T-Damer** [comment]
Okay, thanks then :)

**Comment by backmeupplz** [comment]
> Repetition in this case is a good thing

Sorry, made me laugh out loud :')

**Comment by ljharb** [comment]
There's lots of cases where repetition helps; in test cases, for example.

**Comment by backmeupplz** [comment]
> There's lots of cases where repetition helps

No, there isn't. It's just an exuse to justify WET code. Test cases should also always be DRY.

**Comment by ljharb** [comment]
@backmeupplz that's not my experience. if you DRY things up too much, you die of thirst.

**Comment by ljharb** [comment]
@backmeupplz yes, anyone might be wrong at any time. In this case, I'm not wrong.
